docs/integrations/rsyslog/index.rst

```diff
@@ -0,0 +1,226 @@
+Remote syslog integration
+=========================
+
+In addition to using Aiven for Elasticsearch to store the logs from your
+Aiven services, you can now integrate with an external monitoring system
+that supports the rsyslog protocol.
+
+Creating rsyslog integration
+----------------------------
+
+Add rsyslog integration endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As the first step, you need to add the remote syslog you want to send
+the log to into the project that contains the service you want to
+integrate.
+
+This can be configured from the Service Integrations page in the Aiven
+Console.
+
+.. image:: /images/integrations/remote-syslog-integration_image1.png
+
+Another option is to use the `Aiven
+Client <https://github.com/aiven/aiven-client>`__ .
+
+::
+
+   avn service integration-endpoint-create --project your-project \
+       -d example-syslog -t rsyslog \
+       -c server=logs.example.com -c port=514 \
+       -c format=rfc5424 -c tls=true
+
+When defining the remote syslog server the following parameters can be
+applied
+
+Required:
+
+-  **server** -  DNS name or IPv4 address of the server
+
+-  **port** - port to connect to
+
+-  **format** - message format used by the server, this can be either
+   ``rfc3164`` (the old BSD style message format), ``rfc5424`` (current
+   syslog message format) or custom
+
+-  **tls** - use TLS (as the messages are not filtered and may contain
+   sensitive information, it is highly recommended to set this to true
+   if the remote server supports it)
+
+Conditional (required if format == custom):
+
+-  **logline** - syslog log line template for a custom format,
+   supporting limited rsyslog style templating (using
+   ``%             tag            %`` ). Supported tags are: ``pri`` ,
+   ``timestamp`` , ``timestamp:::date-rfc3339`` , ``HOSTNAME`` ,
+   ``app-name`` , ``procid`` , ``msgid`` , ``msg`` and
+   ``structured-data``
+
+Optional:
+
+-  **sd** - content of the structured data block of ``rfc5424`` message
+
+-  **ca** - (PEM format) Certificate Authority to use for verifying the
+   servers certificate (typically not needed unless the server's
+   certificate is issued by an internal CA or it uses a self-signed
+   certificate)
+
+-  **key** - (PEM format) client key if the server requires client
+   authentication
+
+-  **cert** - (PEM format) client cert to use
+
+Add rsyslog integration to service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This can be configured in the Aiven Console by navigating to the target
+service overview page and then scrolling down to the Manage Integrations
+button.
+
+
+.. image:: /images/integrations/remote-syslog-integration_image2.png
```

Comment:
I'd suggest to remove this image and just add leave it with instructions. You can see step 5 here: https://developer.aiven.io/docs/integrations/datadog/datadog-metrics.html

Reply:
Removed this one as per suggestion.  I have also removed the other old screenshots and replaced with new screenshots from new Aiven console UI.